src/dotnet/App.Maui/MainPage.xaml.cs

```diff
@@ -8,6 +17,27 @@ public partial class MainPage : ContentPage
     private readonly ClientAppSettings _appSettings;
     private readonly NavigationInterceptor _navInterceptor;
 
+    public BlazorWebView BlazorWebView
+        => this._blazorWebView;
+
+#if WINDOWS
```

Comment:
why not put these properties in platform specific files?

Reply:
Good proposal.